### ISSUE TITLE
How to finetune on existing classes without accuracy drop?

### ISSUE THREAD

**Issue by europtripper** [ultralytics/ultralytics#3138]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I want to finetune YOLOv8 on cell phones. This class already exists in pre-trained models. So, I annotated around 275 images on my dataset. But, afterwards the performance got significantly worse on COCO128. 
For annotating my dataset, I annotated with already trained YOLOv8m and then made bounding boxes where I couldn't see that it is able to detect the cell-phones.
I understand that a new class requires roughly 2000 images and 10,000 instances of them, but as this class already exists in COCO's dataset. Why is the accuracy taking a hit that much?

yolov8s.pt        cell phone        128          8    0.795          0.5          0.508     0.317
best.pt             cell phone        128          8    0.00334      0.125      0.064     0.0448

What gives?

![R_curve](https://github.com/ultralytics/ultralytics/assets/136305848/9c9ee90f-ea7f-4dc2-91da-82c05e87293c)
[results.csv](https://github.com/ultralytics/ultralytics/files/11719153/results.csv)
![results](https://github.com/ultralytics/ultralytics/assets/136305848/cc8d342f-2263-40c6-a290-0f95a7ef46bb)
![PR_curve](https://github.com/ultralytics/ultralytics/assets/136305848/bc0f689b-6c8c-4dcf-99fd-1bc8cc208ce6)
![confusion_matrix](https://github.com/ultralytics/ultralytics/assets/136305848/51e99efe-6d32-48b3-bff8-df5dd6909e2e)
![confusion_matrix_normalized](https://github.com/ultralytics/ultralytics/assets/136305848/bd914e06-dcff-4948-abf4-d0ceccf8b292)
![F1_curve](https://github.com/ultralytics/ultralytics/assets/136305848/b939fcfe-1d61-4c36-a8a0-3927a099502e)
![labels](https://github.com/ultralytics/ultralytics/assets/136305848/ddd42ef5-f930-4be7-9f72-0c08d8a77532)
![labels_correlogram](https://github.com/ultralytics/ultralytics/assets/136305848/5d2c1375-3cd5-4f54-b0fc-8385cd36eeac)
![P_curve](https://github.com/ultralytics/ultralytics/assets/136305848/310910a4-d8a5-48b1-af02-cd4a6f82ea13)

**Comment by europtripper** [comment]
From the literature that I read over the last 2 days, it seems that what I am facing is catastrophic forgetting. I implemented freezing the backbone as well as freezing all the layers while gradually unfreezing them. Lowering the LR as well. I don't want to mix a subset of COCO's dataset as of now but let's see.

Also, I was reading about EWC and the performance seems quite good, I will try to see if I am have the time and implement it.

If anyone has other ideas, please feel free to chime in.

**Comment by VyomGarg47** [comment]
Hello @europtripper, I'm sort of facing a similar issue and was curious what worked out best for you. Thanks!

**Comment by bit-scientist** [comment]
@europtripper hi, same as @VyomGarg47 , got quite interested in what you are doing. Could you please explain what you did exactly on this?

> freezing the backbone as well as freezing all the layers while gradually unfreezing them

Thank you!

**Comment by cpingan1201** [comment]
I am facing a similar problem, the forgetting of coco dataset is too significant, but I don't want to add coco dataset in the training set, the parameters like frozen layer and learning rate alone still can't allow me to solve this problem, have you guys found a good solution?

**Comment by Y-T-G** [comment]
@cpingan1201 For detection models, you can't try following this to add an extra head. But this is not meant to perpetually add more classes. Extra head has extra latency.

https://y-t-g.github.io/tutorials/yolov8n-add-classes/
